Makefile

```diff
@@ -42,7 +42,7 @@ reason: setup
 
 .PHONY: clean cleanbisect
 clean: cleanbisect
-	rm -rf _build dune-workspace src/*[0-9]*.ml
+	rm -rf _build dune-workspace
```

Comment:
Don't know why this commit broke the CI build

Reply:
Did you locally merge master into your branch? I suspect there is an interaction between this and #244.